packages/editor-ui/src/composables/usePushConnection.ts

```diff
@@ -529,6 +531,44 @@ export function usePushConnection({ router }: { router: ReturnType<typeof useRou
 		return true;
 	}
 
+	async function trackNodeExecutionFinishedEvent(pushData: PushPayload<'nodeExecuteAfter'>) {
+		const itemCount =
+			pushData.data?.data?.main.reduce((acc, item) => acc + (item?.length ?? 0), 0) ?? 0;
+		const node = workflowsStore.getNodeByName(pushData.nodeName);
+		const status = pushData.data?.executionStatus;
+		const error = pushData.data?.error;
+		let subWorkflowId = node?.parameters.workflowId;
+		let subWorkflowVersion: 'new' | 'legacy' | undefined;
+		if (subWorkflowId) {
+			if (isResourceLocatorValue(subWorkflowId)) {
+				subWorkflowId = subWorkflowId.value;
+			}
+			let subWorkflow: IWorkflowDb | undefined;
+			try {
+				subWorkflow = subWorkflowId
+					? await workflowsStore.getOrFetchWorkflow(String(subWorkflowId))
+					: undefined;
+			} catch {
+				subWorkflow = undefined;
+			}
+			const triggerNode = subWorkflow?.nodes.find(
+				(n) => n.type === EXECUTE_WORKFLOW_TRIGGER_NODE_TYPE,
+			);
+			subWorkflowVersion = triggerNode?.typeVersion === 1.1 ? 'new' : 'legacy';
```

Comment:
```suggestion
			subWorkflowVersion = triggerNode?.typeVersion === 1.0 ?  'legacy' : 'new';
```

so if we update the version we don't change the telemetry too

Reply:
Good catch, I left that for later and then just forgot about it 🙌🏻